jib-cli/src/main/java/com/google/cloud/tools/jib/cli/buildfile/Layers.java

```diff
@@ -47,8 +47,8 @@
    * @return a {@link List} of {@link LayerObject} to use as part of a buildplan
    * @throws IOException if traversing a directory fails
    */
-  static List<LayerObject> toLayers(Path buildRoot, LayersSpec layersSpec) throws IOException {
-    List<LayerObject> layers = new ArrayList<>();
+  static List<FileEntriesLayer> toLayers(Path buildRoot, LayersSpec layersSpec) throws IOException {
```

Comment:
It can be, it doesn't really need to be exposed outside of this package though. Since this isn't a library it doesn't matter all that much. But it really should only be used for conversion within the package.